packages/core/utils/src/index.ts

```diff
@@ -298,3 +304,23 @@ export function isConstructor(obj) {
 export function isEs6Module(module) {
   return (typeof module === 'function' && isConstructor(module)) || (typeof module === 'object' && typeof module.default === 'function' && module.__esModule);
 }
+
+export function warnIfPackageNotDefinedLocally(packageName, warnFunc) {
+  const packageIsResolvable = findUp.sync("node_modules/" + packageName, {cwd: dappPath()});
+  if (!packageIsResolvable) {
+    return warnFunc("== WARNING: "  + packageName + " could not be resolved. ensure that is defined in your dapp's package.json and npm or yarn install was done; In future versions of embark this package should be a local dependency and configured as a plugin");
```

Comment:
```suggestion
    return warnFunc(__("== WARNING: %s could not be resolved. Ensure that it is defined in your Dapp's package.json and npm or yarn install was done;\n In future versions of Embark, this package should be a local dependency and configured as a plugin", packageName));
```
Not 100% sure if the `\n` will work, so might be worth testing locally first.